### PR TITLE
L2 Chain RPC URL validation

### PR DESCRIPTION
**Pull request by mdomasevicius**
This validation is needed for advanced settings in NodeUI

### CODE REVIEW

File: tequilapi/endpoints/validation.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+		return
+	}
+
+	for _, rpc := range rpcURLS {
+		ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+		defer cancel()
+
+		dialContext, err := ethclient.DialContext(ctx, rpc)
+		if err != nil {
+			utils.SendError(resp, err, http.StatusBadRequest)
```
Comment (tomasmik) [inline tequilapi/endpoints/validation.go:62]: There are more 500 like error probably

File: tequilapi/endpoints/validation.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
```
Comment (tomasmik) [inline tequilapi/endpoints/validation.go:2]: it's 2021 🎉

**Review by tomasmik — APPROVED** [review]

File: tequilapi/endpoints/validation_test.go [hunk 3]
```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
```
Comment (mdomasevicius) [inline tequilapi/endpoints/validation_test.go:2]:
```suggestion
 * Copyright (C) 2021 The "MysteriumNetwork/node" Authors.
```

**Review by vkuznecovas — APPROVED** [review]

File: tequilapi/endpoints/validation.go [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+		}
+
+		rpcURLChainID, err := dialContext.ChainID(ctx)
+		if err != nil {
+			utils.SendError(resp, err, http.StatusInternalServerError)
+			return
+		}
+		chain2ID := config.GetInt64(config.FlagChain2ChainID)
+
+		if rpcURLChainID.Cmp(big.NewInt(chain2ID)) != 0 {
```
Comment (vkuznecovas) [inline tequilapi/endpoints/validation.go:73]:
```suggestion
		if chain2ID == rpcURLChainID.Int64() {
```

**Pull request merged**